src/mapping.jl

```diff
@@ -2,6 +2,12 @@
 struct UnWeighted end
 # Weighted mode
 struct Weighted end
+# TriangularWeighted mode
+struct TriangularWeighted end
+
+# Get spacing value based on mode
+get_spacing(::Union{UnWeighted, Weighted}) = 4
+get_spacing(::TriangularWeighted) = 14
```

Comment:
It is so big. Can we make it smaller, just use different spacing for X and Y axis.